source/upgrade/extended-support-release.rst

```diff
@@ -108,15 +104,20 @@ Earlier 4.x versions of Mattermost Desktop App and earlier v1.x versions of the
 
 Please review the `Desktop App changelog </install/desktop-app-changelog.html>`_ and the `Mobile App changelog </deploy/mobile-app-changelog.html>`_ notes for any self-hosted version requirements for features and functionalities, as well as notes on security fixes.
 
-+-------------+------------------+-----------------------------------------+----------------------------------------+
-| ESR Version | Release Date     | Desktop App Minimum Supported Version   | Mobile App Minimum Supported Version   |
-+=============+==================+=========================================+========================================+
-| 7.1         | July 15, 2022    | 5.1.0                                   | 1.54.0                                 |
-+-------------+------------------+-----------------------------------------+----------------------------------------+
-| 6.3         | January 16, 2022 | 5.0.0                                   | 1.48.0                                 |
-+-------------+------------------+-----------------------------------------+----------------------------------------+
-| 5.37        | July 16, 2021    | 4.7.0                                   | 1.45.0                                 |
-+-------------+------------------+-----------------------------------------+----------------------------------------+
++-------------+------------------+-----------------------------------------+------------------------------------------+
+| ESR Version | Release Date     | Desktop App Supported Version           | Mobile App Supported Versions            |
```

Comment:
The way this table is structured doesn't capture the support policy in my mind. I would defer to @nab-77 here and propose we separate this into a different PR so he can take ownership of it and it doesn't block the rest of the stuff in this PR.

My 2c here: Every new mobile release supports the ESRs at that time. That means when an ESR goes out of support, the next mobile release doesn't support it anymore. Hence, there is a **maximum** mobile version that is compatible with each ESR. So really the column for mobile versions should look like this:

v7.1 > Max supported mobile version = v2.0+ (no real max yet since v7.1 hasn't gone out of support yet)
v6.3 > Max supported mobile version = v1.55 (Mobile ESR)
v5.37 > Max supported mobile version = v1.50

I'm sure desktop is the same since it follows the same support policy as mobile. Again, I would defer to @nab-77 here. 
We don't test outdated/old mobile apps against newer server versions, so we can't state the minimum mobile app version compatible with a server

Reply:
Thanks, @esethna. @nab-77 and I have elected to pull this table from the v7.7 docs updates based on your input. We'll revisit together and publish the updated version in a separate PR.